CoursesManager.UI/App.xaml.cs

```diff
@@ -51,13 +51,16 @@ protected override void OnStartup(StartupEventArgs e)
 
         MessageBroker.Subscribe<ApplicationCloseRequestedMessage, App>(ApplicationCloseRequestedHandler, this);
 
-        NavigationService.NavigateTo<StudentManagerViewModel>();
 
         MainWindow mw = new()
         {
             DataContext = new MainWindowViewModel(NavigationService, MessageBroker)
+            
         };
+        GlobalCache.Instance.Put("MainViewModel", mw.DataContext, true);
```

Comment:
Ik dnek we moeten wachten voor dit aanpassen om te bespreken hoe gaan we het injecteren 
Daar hadden we andere logica je mag het zien in mijn PR